gffquant/annotation/count_writer.py

```
@@ -8,32 +8,26 @@
         filtered_readcount=None,
     ):
         self.out_prefix = prefix
-        self.has_ambig_counts = has_ambig_counts
         self.strand_specific = strand_specific
-        self.publish_reports = [
-            item for item in CountWriter.COUNT_HEADER_ELEMENTS
-            if restrict_reports is None or item in restrict_reports
-        ]
-        if report_category:
-            self.publish_reports.append("category")
+        self.publish_reports = list(CountWriter.COUNT_HEADER_ELEMENTS) + ["category",]
```

Ruff (0.16.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)